docs/2.7.0/docs/query/pqs-user-guide.rst

```diff
@@ -0,0 +1,657 @@
+      at zio.Unsafe$.unsafe(Unsafe.scala:37)
+      at zio.ZIOCompanionVersionSpecific.succeed$$anonfun$1(ZIOCompanionVersionSpecific.scala:185)
+      Suppressed: java.net.ConnectException: Connection refused
+        at java.base/sun.nio.ch.Net.pollConnect(Native Method)
+        at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
+        at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:547)
+        at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:602)
+        at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
+        at java.base/java.net.Socket.connect(Socket.java:633)
+        at org.postgresql.core.PGStream.createSocket(PGStream.java:243)
+        at org.postgresql.core.PGStream.<init>(PGStream.java:98)
+        at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:132)
+        at
+          org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:258)
+        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)
+        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:263)
+        at org.postgresql.Driver.makeConnection(Driver.java:443)
+        at org.postgresql.Driver.connect(Driver.java:297)
+        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
+        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:190)
+        at zio.jdbc.shims.postgres$.$anonfun$1(postgres.scala:21)
+        at
+          zio.ZIOCompanionVersionSpecific.attempt$$anonfun$1(ZIOCompanionVersionSpecific.scala:103)
+        at zio.ZIO$.suspendSucceed$$anonfun$1(ZIO.scala:4589)
+        at
+          zio.UnsafeVersionSpecific.implicitFunctionIsFunction$$anonfun$1(UnsafeVersionSpecific.scala:27)
+        at zio.Unsafe$.unsafe(Unsafe.scala:37)
+        at
+          zio.ZIOCompanionVersionSpecific.succeed$$anonfun$1(ZIOCompanionVersionSpecific.scala:185)
+    org.postgresql.util.PSQLException: Connection to localhost:5432 refused. Check that
+    the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
+    java.net.ConnectException: Connection refused
+
+To fix this, make sure that the database exists and is accessible from where you are running the PQS. Also, ensure that the database username and password are correct and that the credentials to connect to the database from the network address are set properly.
+
+If the database connection is broken while the transaction stream was already running, you will see a similar message in the logs, but it will be repeated. The transaction stream will be restarted with an exponential backoff. This gives the database, network or any other troubled resource to get back into shape. Once everything is in order, the stream will continue without any need for manual intervention.
```

Comment:
```suggestion
If the database connection is broken while the transaction stream was already running, you will see a similar message in the logs, but it will be repeated. The transaction stream will be restarted with an exponential backoff. This gives the database, network, or any other troubled resource time to get back into shape. Once everything is in order, the stream will continue without any need for manual intervention.
```